kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/KafkaVersion.java

```diff
@@ -1,24 +1,21 @@
 package com.provectus.kafka.ui.util;
 
-import lombok.extern.slf4j.Slf4j;
+import java.util.Optional;
 
-@Slf4j
 public final class KafkaVersion {
 
   private KafkaVersion() {
```

Comment:
lombok `NoArgsConstructor(access = PRIVATE)` could be used instead or even `UtilityClass`

Reply:
We forbid the usage of UtilityClass:
https://github.com/provectus/kafka-ui/issues/3423#issue-1604463337